### PR TITLE
feat(TUP-25346):tELTTeradataOutput - aliases are mandatory for calculated columns

### PR DESCRIPTION
**Pull request by hcyi**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TUP-25346

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by kjwang-talend — APPROVED** [review]

File: test/plugins/org.talend.designer.dbmap.test/src/org/talend/designer/dbmap/language/mssql/MssqlGenerationManagerTest.java [hunk 1]
```diff
@@ -1110,4 +1109,38 @@ public void testELTMapCrossJoin() {
                 + "\nWHERE" + "\n  B.id = A.id\n  AND  C.id = B.id\"";
         assertEquals(expectedQuery, query);
     }
+
+    @Test
+    public void testBuildSqlSelectWithColumnsAliasIfChecked() {
```
Comment (wchen-talend) [inline test/plugins/org.talend.designer.dbmap.test/src/org/talend/designer/dbmap/language/mssql/MssqlGenerationManagerTest.java:1114]: The alias feature is done only for teradata  , why you add junit for MSSQL ,MYSQL with alias support ?

File: main/plugins/org.talend.designer.components.localprovider/components/tELTTeradataOutput/tELTTeradataOutput_main.javajet [hunk 2]
```diff
@@ -27,19 +37,23 @@ imports="
...
-	           }else{
-	             int index = content.indexOf(".");
-	             if(index != -1){
-	                 result = "src" + content.substring(index);
-	             }else{
-	                 result = content;
-	             }
-	           }
-	           return result;
+	          <% if(!useAlias){%>
```
Comment (wwang-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tELTTeradataOutput/tELTTeradataOutput_main.javajet:40]: better to correct the format

**Review by wwang-talend — COMMENTED** [review]
if USE_ALIAS_IN_OUTPUT_TABLE's default value is false, LGTM now

**Review by wwang-talend — APPROVED** [review]

File: test/plugins/org.talend.designer.dbmap.test/src/org/talend/designer/dbmap/language/mssql/MssqlGenerationManagerTest.java (hunk 1 above)
Comment (hcyi) [reply test/plugins/org.talend.designer.dbmap.test/src/org/talend/designer/dbmap/language/mssql/MssqlGenerationManagerTest.java]: removed

**Review by wchen-talend — APPROVED** [review]

**Pull request merged**